config/prow/cluster/monitoring/mixins/lib/config.libsonnet

```diff
@@ -6,12 +6,30 @@
       'ghproxy.json': 'd72fe8d0400b2912e319b1e95d0ab1b3',
       'slo.json': 'ea313af4b7904c7c983d20d9572235a5',
     },
+    // Component name constants
     components: {
-      tide: 'Tide'
+      // Values should be lowercase for use with prometheus 'job' label.
+      crier: 'crier',
+      deck: 'deck',
+      ghproxy: 'ghproxy',
+      hook: 'hook',
+      horologium: 'horologium',
+      monitoring: 'monitoring', // Aggregate of prometheus, alertmanager, and grafana.
+      prowControllerManager: 'prow-controller-manager',
+      sinker: 'sinker',
+      tide: 'tide',
     },
-    local components = self.components,
+    local comps = self.components,
+
+    // SLO compliance tracking config
     slo: {
-      components: [components.tide],
+      components: [
+        comps.deck,
```

Comment:
LGTM overall, a quick question, are these values defined somewhere else?

Reply:
The component constants are defined directly above (line 10 with a reference for convenience on line 22).